packages/cli/lib/prompts/sandboxesPrompt.js

```diff
@@ -13,8 +20,8 @@ const mapSandboxAccountChoices = portals =>
     .filter(p => isSandbox(p))
     .map(p => {
       return {
-        name: uiAccountDescription(p.portalId, false),
-        value: p.name || p.portalId,
+        name: uiAccountDescription(getAccountIdentifier(p), false),
+        value: p.name || p.portalId || p.accountId,
```

Comment:
Are we able to use the new `getAccountIdentifier()` util here?

Reply:
https://github.com/HubSpot/hubspot-cli/pull/1177/commits/fe36cf45f176212ab281ac82be4080809b85b48d